### PR TITLE
create utils file for shared functions

### PR DESCRIPTION
**Pull request by morgan-adams**
Started utils file for reusable components, more to be added later.

### CODE REVIEW

File: src/most-freq-words.py [hunk 1]
```diff
@@ -4,33 +4,16 @@
...
-    user_dir = None
-    for directory in os.listdir(inbox):
-        if fnmatch.fnmatch(directory, f'{user_name}*'):
-            print(f'User match found in directory: {directory}')
-            user_dir = directory
-            break
-    if user_dir is None:
-        print('No match found.')
-        return
+def most_frequent_words():
```
Comment (shoyo-inokuchi) [inline src/most-freq-words.py]: We prob want to handle user input at a higher-level and pass the input down to this function eventually!

File: src/utils.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    """Return relative path of specified conversation as a string.
+
+    Parameters:
+    inbox_path (string) -- path to directory containing conversations
+
+    Returns:
+    inbox (string) -- path to folder of conversation history data
+    """
+    if not os.path.isdir(inbox_path):
+        # TODO: Declare custom exception for more descriptive error handling :^)
```
Comment (shoyo-inokuchi) [inline src/utils.py]: 👍

File: src/utils.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        raise Exception('Could not find messages directory.')
+
+    # TODO: handle multiple conversations matching user query
```
Comment (shoyo-inokuchi) [inline src/utils.py]: 👍

File: src/utils.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    conv_name = input('Name of message partner (or group chat name): ')
+    conv_name = conv_name.lower().replace(" ", "")
+    for dir in os.listdir(inbox_path):
+        if fnmatch.fnmatch(dir, f'{conv_name}*'):
+            print(f'Conversation match found in directory: {dir}')
+            # TODO: consider returning absolute path instead of relative
```
Comment (shoyo-inokuchi) [inline src/utils.py]: 👍

File: src/most-freq-words.py (hunk 1 above)
Comment (morgan-adams) [reply src/most-freq-words.py]: 😋💦

**Pull request merged**